fastlane/metadata/steam/czech/achievements.vdf

```diff
@@ -0,0 +1,109 @@
+"lang"
+{
+	"Language"	"czech"
+	"Tokens"
+	{
+		"NEW_ACHIEVEMENT_20_0_NAME"	"Ověřen"
+		"NEW_ACHIEVEMENT_20_0_DESC"	"Dokončena výuka."
+		"NEW_ACHIEVEMENT_20_1_NAME"	"Šroťák"
+		"NEW_ACHIEVEMENT_20_1_DESC"	"Znič 1000 nepřátelských jednotek."
+		"NEW_ACHIEVEMENT_20_2_NAME"	"Čistič"
+		"NEW_ACHIEVEMENT_20_2_DESC"	"Znič 100,000 nepřátelských jednotek."
+		"NEW_ACHIEVEMENT_20_3_NAME"	"Přesun do atmosféry"
+		"NEW_ACHIEVEMENT_20_3_DESC"	"Vyšli zpět celkem 10.000 předmětů."
+		"NEW_ACHIEVEMENT_20_5_NAME"	"Nekonečné dodávky"
+		"NEW_ACHIEVEMENT_20_5_DESC"	"Vyšli zpět celkem 1.000.000 předmětů."
+		"NEW_ACHIEVEMENT_20_6_NAME"	"Dobyvatel"
+		"NEW_ACHIEVEMENT_20_6_DESC"	"Vyhraj 10 zápasů v ražimu útoku."
+		"NEW_ACHIEVEMENT_20_7_NAME"	"Šampión"
+		"NEW_ACHIEVEMENT_20_7_DESC"	"Vyhraj 10 zápasů hráčů proti sobě (PvP)."
+		"NEW_ACHIEVEMENT_20_8_NAME"	"Blesk"
+		"NEW_ACHIEVEMENT_20_8_DESC"	"Znič nepřátelské jádro v útočné zóně v průběhu prvních 5 vln."
+		"NEW_ACHIEVEMENT_20_9_NAME"	"Prší jádra"
+		"NEW_ACHIEVEMENT_20_9_DESC"	"Vyšli Tvoje jádro do mapy 30x."
+		"NEW_ACHIEVEMENT_20_10_NAME"	"Houževnatý"
+		"NEW_ACHIEVEMENT_20_10_DESC"	"Přežij 100 vln."
+		"NEW_ACHIEVEMENT_20_11_NAME"	"Nepostvrněný"
+		"NEW_ACHIEVEMENT_20_11_DESC"	"Přežij 500 vln."
+		"NEW_ACHIEVEMENT_20_12_NAME"	"Výzkumník"
+		"NEW_ACHIEVEMENT_20_12_DESC"	"Vyzkoumej všechno."
+		"NEW_ACHIEVEMENT_20_13_NAME"	"Měňavec"
+		"NEW_ACHIEVEMENT_20_13_DESC"	"Odemkni každého mech ve hře a transformuj se do něj."
+		"NEW_ACHIEVEMENT_20_14_NAME"	"Přetížení"
+		"NEW_ACHIEVEMENT_20_14_DESC"	"Zasáhni nepřítele pokrytého vodou energetickou zbraní."
+		"NEW_ACHIEVEMENT_20_15_NAME"	"Odražení"
+		"NEW_ACHIEVEMENT_20_15_DESC"	"Znič jednotku její vlastní odraženou kulkou."
+		"NEW_ACHIEVEMENT_20_17_NAME"	"Distributor v zácviku"
+		"NEW_ACHIEVEMENT_20_17_DESC"	"Vynalezni Směrovač."
+		"NEW_ACHIEVEMENT_20_18_NAME"	"Stvořitel"
+		"NEW_ACHIEVEMENT_20_18_DESC"	"Umísti 10.000 bloků."
+		"NEW_ACHIEVEMENT_20_19_NAME"	"Viktor čistič"
+		"NEW_ACHIEVEMENT_20_19_DESC"	"Znič 1.000 nepřítelských bloků."
+		"NEW_ACHIEVEMENT_20_20_NAME"	"Velkolepá chyba"
+		"NEW_ACHIEVEMENT_20_20_DESC"	"Způsob přetížení Thoriového reaktoru, následované explozí."
+		"NEW_ACHIEVEMENT_20_21_NAME"	"Kartograf"
+		"NEW_ACHIEVEMENT_20_21_DESC"	"Vytvoř 10 nových map."
+		"NEW_ACHIEVEMENT_20_22_NAME"	"Sosač"
+		"NEW_ACHIEVEMENT_20_22_DESC"	"Stáhni mapu z dílny."
+		"NEW_ACHIEVEMENT_20_23_NAME"	"Tvůrce"
+		"NEW_ACHIEVEMENT_20_23_DESC"	"Publikuj mapu do dílny."
+		"NEW_ACHIEVEMENT_20_24_NAME"	"Řezník"
+		"NEW_ACHIEVEMENT_20_24_DESC"	"Poraž Záporáka."
+		"NEW_ACHIEVEMENT_20_25_NAME"	"Průzkumník"
+		"NEW_ACHIEVEMENT_20_25_DESC"	"Odemkni všechny zóny v kampani."
+		"NEW_ACHIEVEMENT_20_26_NAME"	"Perfektionalista"
```

Comment:
perfekCionalista (s T je to podle všeho polsky :D)

Reply:
jj, díky za postřeh